api/src/main/java/org/openmrs/module/openconceptlab/Subscription.java

```diff
@@ -86,11 +94,11 @@ public boolean isManual() {
 		return (days == null && hours == null && minutes == null) || (days == 0 && hours == 0 && minutes == 0);
 	}
 
-	public boolean isSubscribedToSnapshot() {
+	public Boolean isSubscribedToSnapshot() {
```

Comment:
"is" -> always returns boolean and not Boolean, "get" -> returns Boolean (it's the POJO contract), please revert it back.